docs/extensions.md

```diff
@@ -55,6 +55,13 @@ extensions:
     packages:
         - strace
         - ltrace
+    # Optional additional repos
+    repos:
+      - sooper-repo
+    # Optional additional modules
+    modules:
```

Comment:
For module enablement, it'd be trickier to do properly. So can probably just leave as is (but also with a disclaimer about the globalness).